src/Akeneo/Connectivity/Connection/back/Domain/Settings/Model/Write/Connection.php

```diff
@@ -109,4 +109,9 @@ public function disableAudit(): void
     {
         $this->auditable = false;
     }
+
+    public function hasDataDestinationFlowType(): bool
+    {
+        return FlowType::DATA_DESTINATION === $this->flowType()->__toString();
```

Comment:
```suggestion
        return FlowType::DATA_DESTINATION === (string) $this->flowType();
```

It's cleaner, we should avoid directly calling php magic methods like `__toString()`.